packages/react-core/src/components/SimpleList/SimpleListItem.tsx

```diff
@@ -15,7 +17,7 @@ export interface SimpleListItemProps {
   /** Additional props added to the SimpleList <a> or <button> */
   componentProps?: any;
   /** Indicates if the link is current/highlighted */
-  isCurrent?: boolean;
```

Comment:
Isn't this a breaking change? I see this property being used in the existing examples. Thinking it needs to be deprecated instead?

```
<SimpleListItem key="item1" isCurrent>
  List item 1
</SimpleListItem>,
```

Reply:
ok, I can do that, but I think that shouldn't be necessary for the beta component.

Reply:
@wojta I think you make a good point here. If it is a Beta component then I believe backwards compatibility goes out the window.  people should use Beta components at their own risk and expect them to change.